### PR TITLE
fixes #4712 - editing organization gives undefined method parent_taxonomy_selected_location_path

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
this happened with `locations_enabled = false` and `organizations_enabled = true`

File: app/views/taxonomies/_form.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <%= select_f f, :parent_id, taxonomy.class.where("id NOT IN (#{taxonomy.subtree_ids.join(',')})").order(:title), :id, :title, { :include_blank => true },
                { :label => _('Parent'), :onchange => 'parent_taxonomy_changed(this);',
                  :help_inline => :indicator,
-                 :'data-url' => parent_taxonomy_selected_location_path(taxonomy.id)
+                 :'data-url' => (controller_name == 'organizations' ? parent_taxonomy_selected_organization_path(taxonomy.id) :               parent_taxonomy_selected_location_path(taxonomy.id))
```
Comment (domcleal) [inline app/views/taxonomies/_form.html.erb]: lots of whitespace?

Comment (isratrade) [inline app/views/taxonomies/_form.html.erb]: @domcleal, I now have a hard return to link up the two paths.

**Comment by domcleal** [comment]
Merged as 630bb30 for Foreman 1.5.0, thanks @isratrade!

**Pull request closed without merging**